lightning-invoice/src/payment.rs

```diff
@@ -7,7 +7,7 @@
 // You may not use this file except in accordance with one or both of these
 // licenses.
 
-//! A module for paying Lightning invoices.
+//! A module for paying Lightning invoices and sending spontaneous payments.
```

Comment:
I think there's a few other places that need updating? i.e. the docs above `pub struct InvoicePayer` and a few lines down from here where it says it's a "utility for paying invoices"